src/main/java/ava/exception/NoTimeException.java

```diff
@@ -0,0 +1,15 @@
+package ava.exception;
+
+/**
+ * Class to represent Duke.Exception.NoTimeException.
```

Comment:
Ditto with the documentation. There are also other places in the code where this seems to be occurring. Could you perhaps change this as well?